### PR TITLE
Trasect comparison

### PR DESCRIPTION
**Pull request by browniea**
Issue #188  

I added an example notebook for the Delft_3d transect of Tanana.

### CODE REVIEW

**Comment by ssolson** [comment]
Emily we have updated the testing to fix issues on the CI end. Could you pull the latest Develop into your branch to check your PR (same for #190)?

**Comment by ssolson** [comment]
For Thursday
1. Contour plot of Experimental Data
2. Contour plot of  D3D
3. Contour plot of  L1 norm difference
4. Contour plot of L2 Difference
5. On plots 3 and 4 a point indicating the max error (L infinty)

**Comment by ssolson** [comment]
For Tuesday 11/22
1. cleaned experimental data contour
2. comparison plots from the previous post

side quest to look into D3D bathymetry data and if we can get plot/ simulation closer to the city side

**Comment by browniea** [comment]
@ssolson 

I tried using contourf instead of tricontour and I got the error "setting an array element with a sequence. The requested array has an inhomogeneous shape after 1 dimension. The detected shape was (2,) + inhomogeneous part."
I double checked that in inputs to be the same size and regularly spaced. 

I also ran into some problems with the latest Delft3D update. I couldn't open the Tanana model. I'll look into this more after I'm back from break. For now, I'll use the previous plots for the paper.

**Comment by ssolson** [comment]
@browniea the example is no longer in the folder. I think you renamed the file and did not add the new file name to your commit.

**Comment by ssolson** [comment]
Hey @jmcvey3 could you look at the issue above with the `lattitude_gps` error? @browniea let me know she is using a custom Dolfyn version you sent her to process the Teledyne data. We thought this would be included in the recent Dolfyn merge. 

What do we need to do to get this example working for the public?

File: examples/D3d_tanana.ipynb [hunk 1]
```diff
@@ -0,0 +1,1524 @@
...
+     "output_type": "stream",
+     "text": [
+      "C:\\Users\\eabrow\\AppData\\Roaming\\Python\\Python38\\site-packages\\dolfyn\\velocity.py:10: AccessorRegistrationWarning: registration of accessor <class 'dolfyn.velocity.Velocity'> under name 'velds' for type <class 'xarray.core.dataset.Dataset'> is overriding a preexisting attribute with the same name.\n",
+      "  class Velocity():\n"
+     ]
+    }
+   ],
+   "source": [
+    "from mhkit import dolfyn as dlfn\n",
+    "from dolfyn.adp import api\n",
```
Comment (jmcvey3) [inline examples/D3d_tanana.ipynb:20]: This is probably the error. "mhkit.dolfyn.adp"

Comment (browniea) [reply examples/D3d_tanana.ipynb:20]: Thanks! I got it to work.

Comment (ssolson) [reply examples/D3d_tanana.ipynb:20]: Thank you @jmcvey3

**Review by ssolson — APPROVED** [review]

**Pull request merged**